lib/ansible/parsing/yaml/constructor.py

```diff
@@ -127,32 +133,16 @@ def _node_position_info(self, node):
         return (datasource, line, column)
 
 
-AnsibleConstructor.add_constructor(
-    u'tag:yaml.org,2002:map',
-    AnsibleConstructor.construct_yaml_map)
-
-AnsibleConstructor.add_constructor(
-    u'tag:yaml.org,2002:python/dict',
-    AnsibleConstructor.construct_yaml_map)
+AnsibleConstructor.add_constructor(u'tag:yaml.org,2002:map', AnsibleConstructor.construct_yaml_map)
```

Comment:
Did you change anything in these lines? It's kinda hard to easily visually identify whether something is changed...

Reply:
Well, the final code is easier to read on a single line, so rather than extending it the result is IMO more readable so worth the effort.

Reply:
I'd argue that it's less readable because line is too long now